CHANGELOG.md

```diff
@@ -4,9 +4,10 @@
 
 ### New Features
 
-#### Common I/O Library V1.0.0
-- Common I/O library provides a set of standard APIs, across supported reference boards, for accessing common serial devices. 
-- Supported peripherals include UART, SPI and I2C.
+#### Common I/O Library V0.5.0
+- A preview of Common I/O library is included.
+- Common I/O library provides a set of standard APIs, across supported reference boards, for accessing common devices.
+- It will start with supporting serial peripherals for UART, SPI and I2C. Other peripheral support will follow.
```

Comment:
"This preview supports UART, SPI and I2C. Support for other peripherals will follow."

Reply:
> Can you please change the capitalization of the description text to:
> 
> "Update change log for Nuvoton Ethernet and Common I/O library."

Done.

Reply:
OK.